README.md

```diff
@@ -96,196 +62,112 @@ For more information on the importance of a professional README for your reposit
   </ul>
 </details>
 
-<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- Added Item class in item.rb
+- Added Book class to book.rb
+- Preserved data in json files
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
+<!-- GETTING STARTED -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
+### Video Intro
 
-> Add a link to your deployed project.
+A short description of the project:
 
-- [Live Demo Link](https://google.com)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- GETTING STARTED -->
+[Video Intro](https://drive.google.com/file/d/1YMkNOQr9J4tsz6oUlM4wwOfyRlOjXi5U/view?usp=sharing)
 
```

Comment:
- [ ] Great job recording a video, but one requirement was not followed. The maximum video length should be 5. Kindly record another video and try as much as possible to meet all the video requirements.


![video](https://github.com/danielochuba/catalog-of-my-things/assets/53124548/176b7da5-8b65-4e4d-aeb6-60a6a308d700)